### ISSUE TITLE
Interventions completed should be saved

### ISSUE THREAD

**Issue by Kowkutla4** [iupui-soic/eceb#14]
![0-60min](https://user-images.githubusercontent.com/36452137/53767683-8812c180-3ea4-11e9-9f8a-f0e0489e24bd.PNG)
Once the interventions are completed and checked by the user, the changes made should be saved even after going to the next intervention.

**Comment by SherriBucher** [comment]
Yes. Should be saved. And if possible, the check mark does not turn green until/unless all the evidence-based interventions are marked.  If none of interventions are ticked, check mark is red. If some are ticked, check mark is yellow.  

For interventions such as temperature or breathing, if we could also carry forward those prior measurements, from assessment to assessment, in order for the health care providers to see what the progression of the vital signs are over time (and across nursing shifts), that would be very useful.

Please note that, efforts of others to develop a digital partograph (tracking of labor progress over time), might be very useful to our efforts to develop tracking of newborn indicators/activities over time.  Please see this link for background information -- https://static.sched.com/hosted_files/2017ict4dconference/79/ICT4D2017%20Digital%20Partograph%20Presentation.pdf
